<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form_Add</class>
 <widget class="QWidget" name="Form_Add">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>609</width>
    <height>537</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="laber_TEXT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>黑体</family>
     <pointsize>22</pointsize>
    </font>
   </property>
   <property name="text">
    <string>添加联系人：</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>60</y>
     <width>491</width>
     <height>451</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>学号：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_ID"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>姓名：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_Name"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>性别：  </string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="Rbutton_Man">
          <property name="text">
           <string>男</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QRadioButton" name="RButton_Womon">
          <property name="text">
           <string>女</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>年龄：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_Age"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>手机号码：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_Tell"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>家庭地址：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_Address"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_10">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>QQ号码：  </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_QQNum"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_11">
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>黑体</family>
            <pointsize>14</pointsize>
           </font>
          </property>
          <property name="text">
           <string>专业：    </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="Edit_Major"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <spacer name="verticalSpacer_2">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Button_confirm">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>确认</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="Button_Cancel">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>80</height>
         </size>
        </property>
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="verticalSpacer_3">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
